### PR TITLE
Add Chrome Sessions and Tabs

### PR DESCRIPTION
**Pull request by sydp**
This PR adds Chrome/Chromium window sessions and tabs

### CODE REVIEW

File: data/webbrowser.yaml [hunk 1]
```diff
@@ -799,6 +799,52 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Configuration']
 ---
+name: ChromeSessions
+doc: |
+  Google Chrome, Canary and Chromium Sessions files
```
Comment (joachimmetz) [inline data/webbrowser.yaml:804]: Sessions files => Sessions and Tabs files

File: data/webbrowser.yaml [hunk 2]
```diff
@@ -799,6 +799,52 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Configuration']
 ---
+name: ChromeSessions
```
Comment (joachimmetz) [inline data/webbrowser.yaml:802]: ChromeSessions => ChromeSessionsAndTabs

Comment (joachimmetz) [reply data/webbrowser.yaml:802]: This definition also seems to overlap with ChromeSessionStorage

**Review by joachimmetz — CHANGES_REQUESTED** [review]
Please consolidate ChromeSessionStorage and the new definition

File: data/webbrowser.yaml (hunk 2 above)
Comment (sydp) [reply data/webbrowser.yaml]: Fixed a typo - there shouldn't be any overlap with the ChromeSessionStorage definitions now.

**Comment by sydp** [comment]
> Please consolidate ChromeSessionStorage and the new definition

Ok have merged the two definitions.

File: data/webbrowser.yaml [hunk 3]
```diff
@@ -799,9 +799,9 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Configuration']
 ---
-name: ChromeSessions
+name: ChromeSessionsAndTabs
```
Comment (joachimmetz) [inline data/webbrowser.yaml]: add `ChromeSessions` as alias for backwards compatibility

File: data/webbrowser.yaml [hunk 4]
```diff
@@ -833,14 +833,14 @@ sources:
 - type: FILE
   attributes:
     paths:
-    - '%%users.localappdata%%\Chromium\User Data\*\Session Storage\Session_*'
```
Comment (joachimmetz) [inline data/webbrowser.yaml]: are these paths incorrect or from an older version? should the definition contain both?

**Review by joachimmetz — CHANGES_REQUESTED** [review]
Some questions

File: data/webbrowser.yaml (hunk 3 above)
Comment (sydp) [reply data/webbrowser.yaml]: I removed this when merging with the existing sessions definition

File: data/webbrowser.yaml (hunk 4 above)
Comment (sydp) [reply data/webbrowser.yaml]: Initially commit was incorrect.  Fixed now.

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**